oss-internship-2020/gdal/raster_to_gtiff/tests.cc

```diff
@@ -0,0 +1,78 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <optional>
+#include <string>
+
+#include "gtest/gtest.h"
+#include "sandboxed_api/sandbox2/util/fileops.h"
+
+#include "gdal_sandbox.h"  // NOLINT(build/include)
+#include "get_raster_data.h"  // NOLINT(build/include)
+#include "gtiff_converter.h" // NOLINT(build/include)
+#include "utils.h"  // NOLINT(build/include)
+
+namespace {
+
+inline constexpr absl::string_view kTempFilePrefix = "temp_data";
+inline constexpr absl::string_view kFirstTestDataPath = 
+    "../testdata/cea.tif";
```

Comment:
pass `PROJECT_BINARY_DIR` with an env var to the test and use a path relative to that
See: `TEST_SRCDIR` in `sandboxed_api/sandbox2/CMakeLists.txt`

Similar for temp file.